data/arrow.go

```diff
@@ -794,6 +794,9 @@ func (frames Frames) MarshalArrow() ([][]byte, error) {
 	bs := make([][]byte, len(frames))
 	var err error
 	for i, frame := range frames {
+		if frame == nil {
+			continue
```

Comment:
Can this produce a sparse array? On the next iteration `i` will be incremented and on the current one we didn't add anything. Shall we use `append` instead of `bs[i]` below?

Reply:
updated to what @kylebrandt suggested